### PR TITLE
Update team page and images

### PR DESCRIPTION
**Pull request by jahlela**

### CODE REVIEW

**Comment by jahlela** [comment]
The build failed due to missing translations. I have added preliminary translations for my bio, but the French ought to be reviewed by @koyanoa as well. 

@morloy The final text and name/image for the rocket should also be reviewed before we added translations. 

Screen Shot 2019-07-02 at 5 30 34 PM

**Comment by jahlela** [comment]
@morloy Will you take a portrait of Luna tomorrow, or should we go through the final review?

File: src/locale/de/messages.po [hunk 1]
```diff
@@ -520,7 +520,7 @@ msgstr "Funktionen"
 
 #: src/pages/features.jsx:70
 msgid "Featuring unlimited share classes, treasury shares, pooled investments, and automatic share numbering"
-msgstr "Mit beliebig vielen Anteilsklassen, Treasury Anteilen, gepoolten Investitionen und automatischer Aktiennummerierung"
+msgstr "Mit beliebig vielen Anteilsklassen, Treasury Anteilen, gepoolten Investitionen und automatischer Aktiennummerierung<<<<<<< HEAD"
```
Comment (morloy) [inline src/locale/de/messages.po]: That doesn’t look right 😉

File: src/locale/en/messages.po [hunk 2]
```diff
@@ -797,7 +801,11 @@ msgstr "Log In"
 msgid "Lost track of who owns how many shares in your startup? Let Ledgy deal with it."
 msgstr "Lost track of who owns how many shares in your startup? Let Ledgy deal with it."
 
-#: src/pages/about-us.jsx:189
+#: src/layouts/team.jsx:113
+msgid "Luna is a model of the Saturn V rocket, which has been to space 13 times, and holds the record for largest payload to low Earth orbit. She‘s still under construction - <0>Ready to help?</0>"
```
Comment (morloy) [inline src/locale/en/messages.po]:
```suggestion
msgid "Luna is a model of the Saturn V rocket, which has been to space 13 times, and holds the record for the largest payload to low Earth orbit. She‘s still under construction - <0>Ready to help?</0>"
```

File: src/locale/de/messages.po (hunk 1 above)
Comment (jahlela) [reply src/locale/de/messages.po]: Agreed :) Nice reminder to also grep for `<<<<`, and not just `>>>>`

File: src/locale/en/messages.po (hunk 2 above)
Comment (jahlela) [reply src/locale/en/messages.po]: Changed!

**Pull request merged**